package.json

```diff
@@ -2,24 +2,51 @@
   "name": "kubercade",
   "version": "1.0.0",
   "description": "Serverless containerized social arcade built on Knative.",
-  "main": "index.js",
+  "main": "build/app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "npm run prod",
+    "dev": "ts-node-dev --respawn --transpileOnly ./src/app.ts",
+    "prod": "tsc && node ./build/app.js",
```

Comment:
When I run `npm start` the file is `./build/src/app.js`, is there a reason for this difference?

Reply:
I'm not getting that, when I run `npm start` it executes `tsc && node ./build/app.js` as expected. Where is this happening?

Reply:
It's happening when I run it in the normal Linux terminal - I'm trying to debug deployment to Cloud Run.